uasc/secure_channel.go

```diff
@@ -245,6 +260,7 @@ func (s *SecureChannel) SendAsync(svc interface{}, authToken *ua.NodeID) (resp c
 	s.reqhdr.AuthenticationToken = authToken
 	s.reqhdr.RequestHandle++
 	s.reqhdr.Timestamp = time.Now()
+	s.reqhdr.TimeoutHint = uint32(timeout / time.Millisecond)
```

Comment:
We should probably check for a minimum timeout here to make sure the user didn't put in something too small.  Maybe make minimum `cfg.RequestTimeout`?

Reply:
OK. Sounds reasonable